meshtastic/tests/test_mt_config.py

```
@@ -44,9 +102,197 @@ def test_setattr_compat_alias_sets_new_name_and_warns() -> None:
+@pytest.mark.unit
+@pytest.mark.usefixtures("mt_config_state")
+def test_concurrent_access_thread_safety() -> None:
+    """Concurrent access to deprecation tracking should be thread-safe."""
+    results = []
+
+    def access_deprecated_alias() -> None:
+        try:
+            # Access the deprecated alias (warns once per process)
+            _ = mt_config.tunnelInstance  # pyright: ignore[reportAttributeAccessIssue]
+            results.append("success")
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)